lnwallet/fee_estimator.go

```diff
@@ -711,12 +711,12 @@ func (w *WebAPIFeeEstimator) updateFeeEstimates() {
 	// overloaded, we can exit early and use our default fee.
 	netTransport := &http.Transport{
 		Dial: (&net.Dialer{
-			Timeout: 5 * time.Second,
+			Timeout: 1 * time.Second,
```

Comment:
if users are in a place with high latency, does this mean they may never be able to get a fee estimate? or at least with less success than they would with 5 seconds?

perhaps we should have some sort of exponential backoff?